README.md

```diff
@@ -10,7 +10,7 @@
 A Rust library for retrieving random data from (operating) system source. It is
 assumed that system always provides high-quality cryptographically secure random
 data, ideally backed by hardware entropy sources. This crate derives its name
-from Linux's `getrandom` function, but is cross platform, roughly supporting
+from Linux's `getrandom` function, but is cross platform, aiming to support
```

Comment:
What does *support* mean — that *it works* or that *it's a bug if it doesn't work*? My interpretation is the latter, thus `aiming to` only adds confusion.

Also — there was never a decision *not* to try to support additional targets (which is what #4 is about), hence why this did say *roughly*.

Reply:
I reverted this `README` change